Documentation/General/finalReport/appendix-rtree.tex

```diff
@@ -0,0 +1,40 @@
+\chapter{R-Trees as a database level Nearest Neighbours Algorithm}
+
+In MoodCat, the core function of the rooms is to find songs that fit to your mood. 
+Finding these songs should be autonomous and accurate. 
+In our development we opted to achieve this using a K-Nearest Neighbour (KNN) algorithm.
+
+Such an algorithm finds the K nearest elements from a given point. 
+Consider the room to be a point in 2D space (x axis is the valence vale, the y axis is the arousal value). 
+For example, take the graphic below. 
+The red dot is the room we're finding a song for, the green and blue dots are songs defined with a VA-Vector. 
```

Comment:
Explain where VA-vector stands for. We have told above about valence and arousal, but for someone who doesn't have heard about this befrore. It could be a bit difficult to understand.